frontend/src/metabase/visualizations/components/ChartSettings.jsx

```diff
@@ -286,32 +278,11 @@ class ChartSettings extends Component {
       </SectionContainer>
     );
 
-    const widgetList = visibleWidgets.map(widget => (
-      <ChartSettingsWidget
-        key={widget.id}
-        {...widget}
-        {...extraWidgetProps}
-        setSidebarPropsOverride={setSidebarPropsOverride}
```

Comment:
Great now `setSidebarPropsOverride` is removed, since all sub-settings live in the popover now.